libraries/AP_Parachute/AP_Parachute.h

```diff
@@ -58,10 +61,22 @@ class AP_Parachute {
     /// update - shuts off the trigger should be called at about 10hz
     void update();
 
+    /// emergency_enabled - returns true if emergency parachute is enabled
+    bool emergency_enabled() const { return _emergency_enabled; }
+    
+    /// critical_sink - returns the configured maximum sink rate to trigger emergency release
+    float critical_sink() const { return _critical_sink; }
+
     /// alt_min - returns the min altitude above home the vehicle should have before parachute is released
     ///   0 = altitude check disabled
     int16_t alt_min() const { return _alt_min; }
 
+    /// set_is_flying - accessor to the is_flying flag
+    void set_is_flying(const bool is_flying) { _is_flying = is_flying; }
+
+    // get_sink_rate - gets vehicle sink rate
+    void get_sink_rate(float sink_rate) { _sink_rate = sink_rate; }
```

Comment:
this should be set_sink_rate() not get_sink_rate()

Reply:
Right